token/cli/src/bench.rs

```diff
@@ -19,6 +21,47 @@ use {
     std::{sync::Arc, time::Instant},
 };
 
```

Comment:
Could you put in a PR to the monorepo to add the implementation for `Vec<Arc<dyn Signer>>` and `[Arc<dyn Signer>]`? Please tag me on it too.  Then please add a comment here saying to remove this signer implementation once that PR is available in the SDK.

Reply:
PR is now open: https://github.com/solana-labs/solana/pull/27000